src/rendering/renderers/gl/state/mapWebGLBlendModesToPixi.ts

```diff
@@ -24,6 +24,25 @@ export function mapWebGLBlendModesToPixi(gl: GlRenderingContext): Record<BLEND_M
     blendMap['screen-npm'] = [gl.SRC_ALPHA, gl.ONE_MINUS_SRC_COLOR, gl.ONE, gl.ONE_MINUS_SRC_ALPHA];
 
     blendMap.erase = [gl.ZERO, gl.ONE_MINUS_SRC_ALPHA];
+
+    const isWebGl2 = 'WebGL2RenderingContext' in globalThis && gl instanceof globalThis.WebGL2RenderingContext;
```

Comment:
this one might be an issue - if user explicitly specifies WebGL 1 render, this code here assume it WebGL2